php-frontend/src/main/java/org/sonar/plugins/php/api/cfg/PhpCfgBranchingBlock.java

```diff
@@ -52,8 +53,8 @@ public Tree branchingTree() {
   }
 
   @Override
-  public ImmutableSet<CfgBlock> successors(){
-    return ImmutableSet.of(trueSuccessor, falseSuccessor);
+  public Set<CfgBlock> successors(){
+    return SetUtils.immutableSetOf(trueSuccessor, falseSuccessor);
```

Comment:
Is this an API change?

Reply:
Yes. I think it's acceptable if it's documented.

Reply:
In fact, it's not an API change since this class has a default visibility (restricted to its package).